autogen.pl

```diff
@@ -1423,6 +1430,18 @@ sub replace_config_sub_guess {
         my $extra      = $4;
 
         print("=== Submodule: $path\n");
+        if (index($path, "pmix") != -1 and list_contains("pmix", @disabled_3rdparty_packages)) {
+          print("Disabled - skipping openpmix");
+          next;
+        }
+        if (index($path, "prrte") != -1 and list_contains("prrte", @disabled_3rdparty_packages)) {
+          print("Disabled - skipping prrte");
+          next;
+        }
+        if (index($path, "hwloc") != -1 and list_contains("hwloc", @disabled_3rdparty_packages)) {
```

Comment:
Why is hwloc here?  It's not a submodule.

Reply:
I did it for completeness, but I suppose this is worthless since even though we do have a disable-3rdparty option for hwloc.

Reply:
Then why not libevent :).  I don't have strong feelings on removing hwloc vs. adding libevent, but we should be consistent.